v63004/instream.csh

```diff
@@ -14,21 +14,22 @@
 #-----------------------------------------------------------------------------------------------------------------------------
 # List of subtests of the form "subtestname [author] description"
 #-----------------------------------------------------------------------------------------------------------------------------
-# gtm8909	    [jake]  Tests that <ctrl-c> within the help facility no longer leads to EN0256 error upon exit
```

Comment:
There is now a huge block of prior commits that have been removed inadvertently. Please re-add them.

Reply:
I realize now the comment lines have not been removed. They have been moved to a different place but with some upper case letters changed to lower case.  Never mind this comment. The other comment still needs to be addressed.